content/docs/command-reference/plots/index.md

```diff
@@ -13,31 +13,23 @@ usage: dvc plots [-h] [-q | -v] {show,diff,modify,templates} ...
 
 positional arguments:
   COMMAND
-    show         Generate plot from a metrics file.
-    diff         Plot differences in metrics between commits.
-    modify       Modify display properties of data-series plots (has no effect on image-type plots).
-    templates    Write built-in plots templates to a directory (.dvc/plots by default).
+    show        Generate plots from target files or from `plots`
+                definitions in `dvc.yaml`.
+    diff        Show multiple versions of a plot by overlaying them
+                in a single image.
+    modify      Modify display properties of data-series plots
+                defined in stages (has no effect on image plots).
+    templates   Write built-in plots templates to a directory
+                (.dvc/plots by default).
```

Comment:
`plots` sub-command descriptions

Reply:
This and following suggestions has been used to update https://github.com/iterative/dvc/pull/7972